.github/workflows/release-swift-toolchain-binary-sizes.yml

```diff
@@ -68,6 +79,21 @@ jobs:
 
           "matrix=$Matrix" | Out-File -Encoding utf8 -Append $env:GITHUB_OUTPUT
 
+      - name: Determine Swift toolchain version
+        id: toolchain-version
+        env:
+          TEST_SWIFT_TOOLCHAIN_VERSION: "20240509.3"
```

Comment:
Why is this version hardcoded? Can you add a comment?

Reply:
It's arbitrary. And sure, added a comment in the latest commit.